tapalcatl_server/server.go

```diff
@@ -253,15 +253,28 @@ func (bm *OnDemandBufferManager) Get() *bytes.Buffer {
 func (bm *OnDemandBufferManager) Put(buf *bytes.Buffer) {
 }
 
+func logFatalCfgErr(logger JsonLogger, msg string, xs ...interface{}) {
+	logger.Error(LogCategory_ConfigError, msg, xs...)
+	os.Exit(1)
+}
+
 func main() {
 	var listen, healthcheck, debugHost string
 	var poolNumEntries, poolEntrySize int
 	var metricsStatsdAddr, metricsStatsdPrefix string
 
 	hc := handlerConfig{}
 
+	systemLogger := log.New(os.Stdout, "", log.LstdFlags|log.LUTC|log.Lmicroseconds)
```

Comment:
Yes, this is what gets prefixed onto the line, and is what the agent will pick up to use as the timestamp for the message itself. For all other timings that we use in the "data" of the message, we use millis.